virttest/staging/service.py

```diff
@@ -283,8 +283,8 @@ def set_target_command(target):
             target = convert_systemd_target_to_runlevel(target)
             return ["telinit", target]
         return set_target_command
-    # Do not need reset failed in sys_v style system.
-    elif command == "reset_failed":
+    # Do not need reset failed, mask and unmask in sys_v style system.
+    elif command in ["reset_failed", "mask", "unmask"]:
```

Comment:
OK, without the `--now` the `mask` won't kill the service, therefor on systemv we could just ignore it. Although there is the `--add` and `--del` that sounds a bit like mask/unmask but I can't really tell the little implications it might bring.